Java/app/src/main/java/com/taboola/sdk3example/sdk_native/FeedFragment.java

```diff
@@ -0,0 +1,154 @@
+package com.taboola.sdk3example.sdk_native;
+
+import android.content.Context;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.taboola.android.Taboola;
+import com.taboola.android.listeners.TBLNativeListener;
+import com.taboola.android.tblnative.TBLNativePage;
+import com.taboola.android.tblnative.TBLNativeUnit;
+import com.taboola.android.tblnative.TBLPlacement;
+import com.taboola.android.tblnative.TBLRecommendationRequestCallback;
+import com.taboola.android.tblnative.TBLRecommendationsResponse;
+import com.taboola.android.tblnative.TBLRequestData;
+import com.taboola.sdk3example.R;
+
+import java.util.ArrayList;
+
+import static java.sql.DriverManager.println;
```

Comment:
Why are we doing 'println' everywhere?

Reply:
Taken from kotlin project. changed to Log.d